omod/src/main/java/org/openmrs/module/pihcore/page/controller/children/FindChildrenPageController.java

```diff
@@ -41,14 +44,23 @@ public void controller(PageModel model, UiUtils ui, UiSessionContext uiSessionCo
                 phoneNumber = attribute.getValue();
             }
         }
+        String patientInfo = patient.getPatientIdentifier().getIdentifier() + " - "
+                + patient.getFamilyName() + ", "
+                + patient.getGivenName() + ", "
+                + patient.getGender() + ", "
+                + patient.getBirthdate().toString();
         // pass the mother's info to the baby registration form as a list of property-value in the following format:
         // SECTION.QUESTION.FIELD , this matches the existing format of the registration form
         SimpleObject initialValues = SimpleObject.create(
                 "demographics.mothersFirstNameLabel.mothersFirstName", patient.getGivenName(),
-                "contactInfo.phoneNumberLabel.phoneNumber", phoneNumber
+                "contactInfo.personAddressQuestion.country", patient.getPersonAddress().getCountry(),
+                "contactInfo.phoneNumberLabel.phoneNumber", phoneNumber,
+                "registerRelationships.relationships_mother.relationship_type", motherToChildRelationshipType.getUuid() + "-A",
+                "registerRelationships.relationships_mother.other_person_uuid", patient.getUuid(),
+                "registerRelationships.relationships_mother.other_person_name", patientInfo,
```

Comment:
These are just display field we need to pre-set with data, I'm guessing?

Reply:
Yes, that's correct.